public/app/features/explore/RunButton.tsx

```diff
@@ -36,6 +38,7 @@ export function RunButton(props: Props) {
       onRefresh={() => onRun(loading)}
       noIntervalPicker={!showDropdown}
       primary={true}
+      minWidth={minWidth}
```

Comment:
```suggestion
      width={!isSmall && '108px'}
```

Maybe reuse the already existing width property but only set it when it isn't small. Also, when I inspect the size of the button it should take `107.25px` instead so let's set `108px`.